sdk/identity/identity/src/msal/utils.ts

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as msalNode from "@azure/msal-node";
+import * as msalCommon from "@azure/msal-common";
+import { AccessToken, generateUuid } from "@azure/core-http";
```

Comment:
can we move to using `uuid` directly instead of relying on `generateUuid` from core-http?